include/modem/nrf_modem_lib.h

```diff
@@ -38,27 +38,37 @@ enum nrf_modem_mode {
 };
 
 /**
- * @brief Initialize the Modem library and turn on the modem.
+ * @brief Initialize the Modem library in normal operating mode and turn on the modem.
```

Comment:
I would remove the "normal operating mode" wording and simplify a bit, based on the doc in `nrf_modem.h`.
`Initialize the Modem library and turn on the modem`